src/locale/deviceLocales.ts

```diff
@@ -41,17 +41,19 @@ export function getLocales() {
     }
 
     if (typeof locale.languageTag === 'string') {
-      if (locale.languageTag.startsWith('zh-Hans')) {
-        // Simplified Chinese to zh-CN
-        locale.languageTag = 'zh-CN'
+      if (
+        locale.languageTag === 'zh-CN' ||
+        locale.languageTag.startsWith('zh-Hans')
```

Comment:
There are multiple `zh-Hans` / `zh-Hant` variants on iOS and Android, such as `zh-Hans-SG`. We want to match as many of these language variants as possible.